pkg/controller/kogitoapp/resource/build_config.go

```diff
@@ -44,39 +44,43 @@ const (
 var (
 	// ImageStreamTag default tag version for the ImageStreams
 	ImageStreamTag = version.Version
+	// ImageStreamNameList holds all kogito images, if one of them does not exists, it will be created
+	ImageStreamNameList = [...]string{
+		"kogito-quarkus-ubi8",
+		"kogito-quarkus-jvm-ubi8",
+		"kogito-quarkus-ubi8-s2i",
+		"kogito-springboot-ubi8",
+		"kogito-springboot-ubi8-s2i",
+		"kogito-data-index",
+	}
 )
 
 // BuildImageStreams are the image streams needed to perform the initial builds
 var BuildImageStreams = map[BuildType]map[v1alpha1.RuntimeType]v1alpha1.Image{
 	BuildTypeS2I: {
 		v1alpha1.QuarkusRuntimeType: v1alpha1.Image{
-			ImageStreamName:      "kogito-quarkus-ubi8-s2i",
-			ImageStreamNamespace: ImageStreamNamespace,
-			ImageStreamTag:       ImageStreamTag,
+			ImageStreamName: "kogito-quarkus-ubi8-s2i",
```

Comment:
Instead of duplicating the imageStreamName we can have `const`s and then reference them here and in the splice  `ImageStreamNameList`

Reply:
do you mean create constants for the imagestream names, i.e.:

quarkusS2i = "kogito-quarkus-s2i"

?

Reply:
Yes